name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/MerkurPrivatBankPDFExtractor.java

```diff
@@ -0,0 +1,115 @@
+package name.abuchen.portfolio.datatransfer.pdf;
+
+import static name.abuchen.portfolio.util.TextUtil.trim;
+
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.Block;
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.DocumentType;
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.Transaction;
+import name.abuchen.portfolio.model.BuySellEntry;
+import name.abuchen.portfolio.model.Client;
+import name.abuchen.portfolio.model.PortfolioTransaction;
+
+@SuppressWarnings("nls")
+public class MerkurPrivatBankPDFExtractor extends AbstractPDFExtractor
+{
+
+    public MerkurPrivatBankPDFExtractor(Client client)
+    {
+        super(client);
+        
+        // @formatter:off
+        // Am Marktplatz 10 · 97762 Hammelburg
+        // @formatter:on
+        addBankIdentifier("Am Marktplatz 10 · 97762 Hammelburg"); //$NON-NLS-1$
+
+        addBuySellTransaction();
+    }
+
+    @Override
+    public String getLabel()
+    {
+        return "MERKUR PRIVATBANK KGaA"; //$NON-NLS-1$
+    }
+
+    private void addBuySellTransaction()
+    {
+        DocumentType type = new DocumentType("Kauf");
+        this.addDocumentTyp(type);
+
+        Transaction<BuySellEntry> pdfTransaction = new Transaction<>();
+        pdfTransaction.subject(() -> {
+            BuySellEntry entry = new BuySellEntry();
+            entry.setType(PortfolioTransaction.Type.BUY);
+            return entry;
+        });
+        
+        // Start from: Wertpapier Abrechnung Kauf
+        Block firstRelevantLine = new Block("^Wertpapier Abrechnung Kauf.*$");
+        type.addBlock(firstRelevantLine);
+        firstRelevantLine.set(pdfTransaction);
+        
+        pdfTransaction
+                        // @formatter:off
+                        // Type (Optional)
+                        // Wertpapier Abrechnung Kauf
+                        // @formatter:on
+                        .section("type").optional().match("^Wertpapier Abrechnung (?<type>Kauf).*$").assign((t, v) -> {
+                            if (v.get("type").equals("Kauf"))
+                                t.setType(PortfolioTransaction.Type.BUY);
+                        })
+                        // @formatter:off
+                        // Security identification:
+                        // Stück 125,3258 XTR.(IE) - MSCI WORLD              IE00BJ0KDQ92 (A1XB5U)
+                        // REGISTERED SHARES 1C O.N.
+                        // @formatter:on
+                        .section("name", "ln2", "isin", "wkn")
```

Comment:
Change `ln2` in `name1`

**All importers work with the same attribute values.**
"name or nameContinued

For `nameContinued` there is a special function ([see here](https://github.com/buchen/portfolio/blob/master/name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/AbstractPDFExtractor.java#L189:L192))
But since this is not applicable here, because we first check if the next line is "Handels-/Ausführungsplatz Außerbörslich", `name1` is the correct attribute name.

Reply:
FYI: I changed this as SonarLint was complaining: "String literals should not be duplicated (java:S1192)"
To fix it one should reduce to len < 5 or create a constants.

Anyway, I will adjust as advised!